<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>Rapid host switching in generalist *Campylobacter* strains erodes the signal for tracing human infections</title>
  <meta name="description" content="">
  <meta name="author" content="Bethany Dearlove">
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
  <link rel="stylesheet" href="libraries/frameworks/revealjs/css/reveal.min.css">
  <link rel="stylesheet" href="libraries/frameworks/revealjs/css/theme/serif.css" id="theme">
  <link rel="stylesheet" href="libraries/highlighters/highlight.js/css/zenburn.css" id="theme">
  <!--[if lt IE 9]>
  <script src="lib/js/html5shiv.js"></script>
  <![endif]-->  <link rel="stylesheet" href = "assets/css/ribbons.css">

</head>
<body>
  <div class="reveal">
    <div class="slides">
      <section class='' data-state='' id='slide-1'>
  
  <p><br></p>

<h2><b><small>Rapid host switching in generalist <em>Campylobacter</em> strains erodes the signal for tracing human infections</small></b></h2>

<p><br><br>Bethany Dearlove</p>

<div style="font-size:30px">University of Cambridge</div>

<p><br>
<br>
<span style="font-size:20px"><img src="assets/img/twitter-icon.png" style="background:none; border:none; box-shadow:none; vertical-align:middle"/> &nbsp;<a href="http://www.twitter.com/bethanydearlove" target="_blank" style="vertical-align:middle">@bethanydearlove</a></span></p>

<!---
Notes view:
<aside class='notes'>
  Type your notes here
</aside>
--->

</section>
<section class='' data-state='' id='slide-2'>
  <h3><em>Campylobacter</em></h3>
  <p><br></p>

<ul>
<li>Major cause of bacterial gastroenteritis.

<ul>
<li>Causes severe diarrhoea, vomiting and abdominal pain.</li>
<li>Symptoms last 7-10 days.</li>
<li>Can lead to more serious diseases such as Guillain-Barr√© syndrome and reactive arthritis.</li>
</ul></li>
</ul>

<!--- 
this is a comment
--->

</section>
<section class='' data-state='' id='slide-3'>
  <h3>Increasing incidence</h3>
  <p><img src="assets/fig/unnamed-chunk-1-1.png" alt="plot of chunk unnamed-chunk-1"> </p>

<div style="font-size:16px">Data source: HPE</div>

</section>
<section class='' data-state='' id='slide-4'>
  <h3>Transmission</h3>
  <p><br></p>

<ul>
<li>Common transmission routes:

<ul>
<li>Undercooked meat and unpasteurised milk</li>
<li>Untreated water</li>
</ul></li>
<li>Human-to human-transmission rare</li>
</ul>

</section>
<section class='' data-state='' id='slide-5'>
  <h3>Zoonosis</h3>
  <p><br></p>

<!--- 
(it is also a zoonotic pathogen, with a wide range of wild and domestic hosts, where it carried, apparently asymptomatically in the gut microbiota)
--->

<ul>
<li>Range of wild and domestic hosts</li>
<li>Asymptomatic carriage in the gut microbiota.</li>
<li>Contamination can occur any time from farmyard to fork.
<br><br>
<div style="text-align:center">
<img src='assets/img/zoonosis.svg' width=50% style="background:none; border:none; box-shadow:none; padding-bottom:0; bottom:0; margin: 0 auto;"/>
<div style="font-size:16px;"><img src='assets/img/CC3.png' style="background:none; border:none; box-shadow:none; vertical-align:bottom"> &nbsp;<a href="http://www.snap2objects.com/2009/06/9-free-vector-animal-farm/" target="_blank" style="vertical-align:middle">Mauricio Duque</a></span></div>
</div></li>
</ul>

</section>
<section>
   <section class='' data-state=''>
    <h3>Characterising zoonosis</h3>
    <p><br></p>

<ul>
<li>MLST analyses have shown that:

<ul>
<li>Populations in different host species tend to be genetically isolated, despite shared geography.</li>
<li>However, some of the most common disease-causing strains are isolated from multiple host species.</li>
</ul></li>
</ul>

<!---
Because it doesn't survive well outside the gut..., regardless of shared geography.  and this information can be used to attribute the sources of clinical infections by exploiting differences in the Campylobacter strains living in different reservoirs. 
--->

    <aside class='notes'>
      
    </aside>
   </section>
   <section class='' data-state=''>
    <h3>Characterising zoonosis</h3>

<p><br></p>

<div style="text-align:center">
<img src='assets/img/Sheppard2013a.svg' width=100% style="background:none; border:none; box-shadow:none; padding-bottom:0; bottom:0; margin: 0 auto;" onmouseover="this.src='assets/img/Sheppard2013.svg'" onmouseout="assets/img/Sheppard2013a.svg'" />
<div style="font-size:16px;"><a href="http://www.pnas.org/content/110/29/11923" target="_blank" style="vertical-align:middle">Sheppard et al. (2013). PNAS.</a></div>
</div>

<!---
And we can see that with MLST data. Here is a phylogenetic tree from Sheppard et al. 2013, with the coloured circles representing the host species. These single colour clades are phylogenetically distinct. However, these green lines mark 
--->

    <aside class='notes'>
      
    </aside>
   </section>
</section>
<section class='' data-state='' id='slide-7'>
  <h3>Characterising zoonosis</h3>
  <p><br></p>

<ul>
<li>Genuine generalists?</li>
<li>Just the limited resolution of MLST?</li>
</ul>

<!---
Investigating how Campylobacter is able to transmit between host species is fundamental in informing public health strategies and subsequently reducing the current upwards trend in cases.
--->

</section>
<section class='' data-state='' id='slide-8'>
  <h3>Aim:</h3>
  <p><br>
To investigate the utility of whole-genome sequences for</p>

<p>
<ul style="padding-left: 60px;">
<li>estimating the rate of zoonosis</li>
<li>improving the accuracy of source attribution for clinical cases</li>
</ul>
</p>

<p style="float:left; padding-left:60px">in these clonal complexes.</p>

</section>
<section class='' data-state='' id='slide-9'>
  <h3>Challenges</h3>
  <p><br></p>

<ul>
<li>Zoonosis model</li>
<li>Recombination</li>
<li>Computation </li>
</ul>

</section>
<section>
   <section class='' data-state=''>
    <h3>Modelling zoonosis</h3>
    <p><br></p>

<ul>
<li>Phylogeography model (Lemey et al., 2009)</li>
<li>If adapted in the past, would expect isolates from same host population to cluster together.</li>
</ul>

    <aside class='notes'>
      
    </aside>
   </section>
   <section class='' data-state=''>
    <h3>Phylogeography</h3>

<p><br></p>

<div style='float:left;width:48%;' class='centered'>
  <img src='assets/img/phylogeo_zoo.svg' style="background:none; border:none; box-shadow:none;"/>
  <span style="font-size:16px; vertical-align:bottom"><img src='assets/img/CC3.png' style="background:none; border:none; box-shadow:none; vertical-align:bottom"> &nbsp; <a href="http://www.snap2objects.com/2009/06/9-free-vector-animal-farm/" target="_blank" style="vertical-align:middle">Mauricio Duque</a></span>
  <p>Transitions between  hosts</p>
  <p style="font-size:24px">Human = chicken or pig or cattle</p>
</div>

<p><div style='float:right;width:48%;'>
  <img src='assets/img/phylogeo_nuc.svg' style="background:none; border:none; box-shadow:none;"/>
  <span style="font-size:16px"></span>
  <p>Transitions between nucleotides</p>
  <p style="font-size:24px">N = A or C or G or T</p></p>

<!---
* The ambiguity code will give a posterior probability of each host for each human isolate.
--->

    <aside class='notes'>
      
    </aside>
   </section>
</section>
<section class='' data-state='' id='slide-11'>
  <h3>Accounting for ancestral recombination</h3>
  <p><br></p>

<ul>
<li>Substitution model

<ul>
<li>Relaxed clock</li>
<li>Gamma site heterogeneity</li>
</ul></li>
<li>Removal of homoplasies

<ul>
<li>Hedge and Wilson (2014)</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-12'>
  <h3>ST-45 complex</h3>
  <p><img src='assets/img/ST45-tree.svg' style="background:none; border:none; box-shadow:none; height:100%"/></p>

</section>
<section class='' data-state='' id='slide-13'>
  <h3>ST-21 complex</h3>
  <p><img src='assets/img/ST21-tree.svg' style="background:none; border:none; box-shadow:none; height:100%"/></p>

</section>
<section class='' data-state='' id='slide-14'>
  <h3>ST-828 complex</h3>
  <p><img src='assets/img/ST828-tree.svg' style="background:none; border:none; box-shadow:none; height:100%"/></p>

</section>
<section class='' data-state='' id='slide-15'>
  <h3>Parameter estimates</h3>
  <p><br>
Estimated one jump every...</p>

<ul>
<li>1.6 years in ST-21</li>
<li>1.8 years in ST-45</li>
<li>12 years in ST-828</li>
</ul>

<!---
rejected single jump hypothesis in all STs
robust to alternative prior beliefs
--->

</section>
<section class='' data-state='' id='slide-16'>
  <h3>Parameter estimates</h3>
  <p><br>
<img src='assets/img/table.svg' style="background:none; border:none; box-shadow:none;"/></p>

</section>
<section class='' data-state='' id='slide-17'>
  <h3>Source of clinical cases</h3>
  <p><img src='assets/img/clinical-source.svg' style="background:none; border:none; box-shadow:none; width:100%"/></p>

<!---
89% of cases chicken
consistent with MLST, but still lack in confidence
even when isolates most closely related to clades sampled only from chicken, still uncertainty of 30-40%
--->

</section>
<section class='' data-state='' id='slide-18'>
  <h3>Summary</h3>
  <p><br></p>

<ul>
<li><p>Isolates from different host species are often more closely related than those isolated from the same host species. </p></li>
<li><p>As expected from previous MLST studies, much ancestry is inferred to occur within chickens.</p>

<ul>
<li>MRCA of all three complexes. </li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-19'>
  <h3>Summary</h3>
  <p><br></p>

<ul>
<li><p>Cannot properly attribute at the individual level in these complexes due to weak host signal.</p>

<ul>
<li>Most likely due to generalist nature of these complexes, and uncertainty reflects make up of host population.</li>
<li>Signal could be drowned out or in accessory genome.</li>
</ul></li>
<li><p>Evidence of potential for whole genomes </p>

<ul>
<li>BUT the extra information from WGS does not overcome the need for detailed sampling.</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-20'>
  <h3>Acknowledgements</h3>
  <p><br>
Daniel Wilson</p>

<p>Samuel Sheppard</p>

<p>Ben Pascoe</p>

<p>Guillaume Meric</p>

<p>Alison Cody</p>

<p>Simon Frost</p>

<p><a href='http://www.nature.com/ismej/journal/vaop/ncurrent/abs/ismej2015149a.html' border='0'><img src='https://chart.googleapis.com/chart?cht=qr&chl=http%3A%2F%2Fwww.nature.com%2Fismej%2Fjournal%2Fvaop%2Fncurrent%2Fabs%2Fismej2015149a.html&chs=150x150&choe=UTF-8&chld=L|2' alt='' target="_blank" style="width:130px"></a></p>

</section>
<section class='' data-state='' id='slide-21'>
  
  
</section>
    </div>
  </div>
</body>
  <script src="libraries/frameworks/revealjs/lib/js/head.min.js"></script>
  <script src="libraries/frameworks/revealjs/js/reveal.min.js"></script>
  <script>
  // Full list of configuration options available here:
  // https://github.com/hakimel/reveal.js#configuration
  Reveal.initialize({
    controls: true,
    progress: true,
    history: true,
    center: false,
    theme: Reveal.getQueryHash().theme || 'serif', 
    transition: Reveal.getQueryHash().transition || 'default', 
    dependencies: [
    // Cross-browser shim that fully implements classList -
    // https://github.com/eligrey/classList.js/
      { src: 'libraries/frameworks/revealjs/lib/js/classList.js', condition: function() { return !document.body.classList;}},
      // Zoom in and out with Alt+click
      { src: 'libraries/frameworks/revealjs/plugin/zoom-js/zoom.js', async: true, condition: function() { return !!document.body.classList; } },
      // Speaker notes
      { src: 'libraries/frameworks/revealjs/plugin/notes/notes.js', async: true, condition: function() { return !!document.body.classList; } },
      // Remote control your reveal.js presentation using a touch device
      //{ src: 'libraries/frameworks/revealjs/plugin/remotes/remotes.js', async: true, condition: function() { return !!document.body.classList; } }
      ]
  });
  </script>  <!-- LOAD HIGHLIGHTER JS FILES -->
<script src="libraries/highlighters/highlight.js/highlight.pack.js"></script>
<script>hljs.initHighlightingOnLoad();</script>
<!-- DONE LOADING HIGHLIGHTER JS FILES -->
 

</html>